stable/prometheus-node-exporter/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: 0.18.1
 description: A Helm chart for prometheus node-exporter
 name: prometheus-node-exporter
-version: 1.7.3
+version: 1.7.4
```

Comment:
Since it's a feature, semver dictates that the minor version should be updated:
> Minor version Y (x.Y.z | x > 0) MUST be incremented if new, backwards compatible functionality is introduced to the public API.
https://semver.org/#spec-item-7

Reply:
@vsliouniaev thank you a lot for the review! I bumped the chart version.